spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -946,31 +961,8 @@ def insert_horizontal_ruler(self):
         """
         self._control.insert_horizontal_ruler()
 
-    # ---- Public methods (overrode by us) ------------------------------------
-    def _event_filter_console_keypress(self, event):
```

Comment:
I didn't change this method and the ones below. I just moved them to the appropriate sections (i.e. `Private`, `Private API (overrode by us)`, etc) in this class.